Jenkinsfile

```diff
@@ -163,13 +178,23 @@ pipeline {
       }
     }
 
-    stage('Push Images') {
-      agent { label 'releaser-v2' }
+    stage('Release') {
       // Only run this stage when triggered by a tag
       when { tag "v*" }
-      steps {
-        // The tag trigger sets TAG_NAME as an environment variable
-        sh 'summon -e production ./bin/publish'
+
+      parallel {
+        stage('Push Images') {
+          steps {
+            // The tag trigger sets TAG_NAME as an environment variable
+            sh 'summon -e production ./bin/publish'
+          }
+        }
+        stage('Create draft release') {
+          steps {
+            sh "summon --provider summon-conjur --yaml 'GITHUB_TOKEN: !var github/users/conjur-jenkins/api-token' ./bin/build_release"
+            archiveArtifacts 'dist/goreleaser/'
```

Comment:
Doesn't build already archive the artifacts?

Reply:
Build only runs on master

Reply:
Whereas the complete draft release is on tag